### PR TITLE
fix: start service returns existing running service

### PR DESCRIPTION
**Pull request by zcstarr**
when available

### CODE REVIEW

File: src/lib/serviceManager.ts [hunk 1]
```diff
@@ -63,6 +63,11 @@ export class ServiceManager {
       notifications: new EventEmitter(),
     };
 
+    const svc = this.manager.getService(serviceName, version, env);
+    if (svc && svc.active && svc.active.state === "running") {
+      return svc.active;
```
Comment (shanejonas) [inline src/lib/serviceManager.ts]: this should return a promise, maybe we can move it down?

Comment (zcstarr) [reply src/lib/serviceManager.ts]: it does async automatically returns a promise

Comment (shanejonas) [reply src/lib/serviceManager.ts]: oh. well then. lol

Comment (devonwesley) [reply src/lib/serviceManager.ts]: yeah wraps return in promise

**Pull request merged**